### DISCUSSION TITLE
Shared login . Laravel App and WordPress using Corcel

### DISCUSSION THREAD

**Discussion by kuyawing** [corcel/corcel#630] [General]
Hi, I read through the documentation but still couldn't make a clear answer for myself.

I have a Laravel app with authentication system. I wanted to add a forum installed on subdomain using WordPress but that entails my Laravel users to register again for the forum(i.e. create username and pass). 

Is Corcel the answer for my problem? Meaning my existing Laravel users can use the same username and password to login to the forum built on WordPress? Can also be reverse? Like if I have a WordPress blog first would they able to login with the same credentials to the Laravel app?

**Comment by xujiongze** [comment]
Some new here ?
I already have a wordpress site, and need create a new Laravel site.
And I want user only need to login a time from wordpress, and can access like login user to laravel.
But I can't found information for this.
